osidb/serializer.py

```diff
@@ -216,6 +220,42 @@ class Meta:
         fields = ["created_dt", "updated_dt"]
         abstract = True
 
+    def update(self, instance, validated_data):
+        """
+        set instance updated timestamp to the one in the request
+        so the TrackingMixin can later perform the comparison
+
+        it is not actually used when saving the instance to DB
+        as that timestamp is being obtained automatically
+        """
+        instance.updated_dt = make_aware(
+            datetime.strptime(
+                self.context["request"].data.get("updated_dt"), DATETIME_FMT
+            )
+        )
```

Comment:
I am afraid this won't be so straightforward. `updated_dt` is readOnly field but you are counting on it to be provided here. This is a schema for this current branch and `updated_dt` is not there and thus it is not very clear that this value should be send in the request body. One workaround might be to treat it the similar way as I treated it the `embargoed` in ACL mixin by creating a custom `EmbargoedField`. :thinking: 

![image](https://user-images.githubusercontent.com/43763876/227236371-e6aab236-2be7-4351-a64c-a978e99ddf09.png)